internal/fs/buffer/inmemory_write_buffer.go

```diff
@@ -15,13 +15,19 @@
 package buffer
 
 import (
-	"bytes"
+	"fmt"
 )
 
 type InMemoryWriteBuffer struct {
```

Comment:
Added the header, thanks!
Unsure what you mean by sample code usage. This should be used with the interface public methods. Should I add the comments explaining it?

Reply:
> Unsure what you mean by sample code usage.

Sample code usage to how this function/struct is to be used e.g. https://github.com/GoogleCloudPlatform/gcsfuse/blob/3bab5990e4e377dbd0160087cd0c70cbafbd55c9/internal/storage/gcs/bucket.go#L86 . This just makes it easier to understand how this struct/function should be used.

This can refer to the header of the interface (WriteBuffer), but I see that WriteBuffer also didn't have any sample code for usage.

Reply:
Added to the WriteBuffer interface